### ISSUE TITLE
Group different variants of the same theme

### ISSUE THREAD

**Issue by a5hk** [microsoft/vscode#130936]
If there is more than one variant of the same theme, group them.

So to select `GitHub Dark` one goes through these steps: `Command Palette` > `Preferences: Color Theme` > `GitHub` > `GitHub Dark`. And maybe add a key binding to go back one level to easily try another theme group (I wasn't able to find such a key binding if it already exists.).

This is related to #125766.

**Comment by aeschli** [comment]
We currently don't have such a theme category, just name and type (dark, light...). And there already thousands of themes published. I think its too late for that.

**Comment by a5hk** [comment]
Is it possible to use the `displayName` used in `package.json` of the theme and group themes under that name?

**Comment by aeschli** [comment]
That would be group by extension, then?

**Comment by a5hk** [comment]
Yes exactly, I meant that from the start, Apparently I didn't communicate that well.
